src/tools/clippy/clippy_lints/src/unnecessary_wraps.rs

```diff
@@ -144,7 +144,9 @@ impl<'tcx> LateLintPass<'tcx> for UnnecessaryWraps {
                 (
                     "this function's return value is unnecessary".to_string(),
                     "remove the return type...".to_string(),
-                    snippet(cx, fn_decl.output.span(), "..").to_string(),
+                    // FIXME: we should instead get the span including the `->` and suggest an
+                    // empty string for this case.
+                    "()".to_string(),
```

Comment:
Do you think you'll address this in this PR or do you want to leave this open?

If you want to leave this open, the suggestion using this string should be turned into `HasPlaceholder`

Reply:
The thing is that suggesting `-> ()` and a complete removal are equivalent, just not stylistic. I don't consider addressing that to be critical to landing this PR (particularly because the existing suggestion is flat out wrong).

Reply:
Oh, I see. The suggestion was just wrong before (at least as far as rustfix is concerned). Yeah, this is not critical before landing this PR. :+1: